include/rdma/fabric.h

```diff
@@ -85,6 +85,15 @@ enum {
 #define FI_VERSION_LT(v1, v2)	((FI_MAJOR(v1) < FI_MAJOR(v2)) || \
 				 (FI_MAJOR(v1) == FI_MAJOR(v2) && FI_MINOR(v1) < FI_MINOR(v2)))
 
+#define OFI_OPS(_fid)		(*_fid)->ops
+#define OFI_FID_OPS(_fid)	(*_fid)->fid.ops
+#define OFI_MR_OPS(_fid)	(*_fid)->mr
+#define OFI_CM_OPS(_fid)	(*_fid)->cm
+#define OFI_MSG_OPS(_fid)	(*_fid)->msg
+#define OFI_RMA_OPS(_fid)	(*_fid)->rma
+#define OFI_TAG_OPS(_fid)	(*_fid)->tagged
+#define OFI_ATOM_OPS(_fid)	(*_fid)->atomic
```

Comment:
I'm fine adding macros that will get used by another PR.  But I agree that these don't seem very useful and appear to increase the amount of typing needed.